app/views/conditions/edit.erb

```diff
@@ -0,0 +1,78 @@
+<% @title = "Edit Weather Info" %>
```

Comment:
If we want dynamic html heads, where should we set the variable? The controller?